docs/book/provider_implementations/building_running_and_testing.md

```diff
@@ -0,0 +1,60 @@
+# Building
+
+## Fixup Makefile to include common Cluster API CRDs
+
+Apply the following patch:
+
+```bash
+diff --git a/Makefile b/Makefile
+index ac12c7e..cf44312 100644
+--- a/Makefile
++++ b/Makefile
+@@ -22,12 +22,15 @@ install: manifests
+ 
+ # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+ deploy: manifests
+-	kubectl apply -f config/crds
+-	kustomize build config/default | kubectl apply -f -
++	cat provider-components.yaml | kubectl apply -f -
+ 
+ # Generate manifests e.g. CRD, RBAC etc.
+ manifests:
+ 	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go all
```

Comment:
We switched away from checking in vendor and instead have established a `hack/tools` convention - I'd recommend checking out what we do in CAPI.